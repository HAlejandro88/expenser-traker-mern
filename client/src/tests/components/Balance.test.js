import React from 'react';
import '@testing-library/jest-dom'
import Balance from '../../components/Balance';
import { shallow } from 'enzyme';


describe('pruebas en e componente <Balance/>', () => {
    test('debuelve el componente contra lo que esta en el sanpshot', () => {
        const wrapper = shallow(<Balance/>)
        expect(wrapper).toMatchSnapshot();
    })

    test('pruebas en las etiquetas ', () => {
        const wrapper = shallow(<Balance/>)
        console.log(wrapper.html())
        expect(wrapper.find('h4').text().trim()).toBe('Your Balance')
        expect(wrapper.find('h1').text().trim()).toBe('$420.00')
    })

    test('pruebas en el useContext', () => {
        // hacer pruebas de useContext
    })
    
    
    
})