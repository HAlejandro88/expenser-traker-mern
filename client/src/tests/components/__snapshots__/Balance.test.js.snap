// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en e componente <Balance/> debuelve el componente contra lo que esta en el sanpshot 1`] = `ShallowWrapper {}`;
